automatic/icloud/icloud.nuspec

```diff
@@ -26,7 +26,7 @@ This is a nuspec. It mostly adheres to https://docs.nuget.org/create/Nuspec-Refe
     <!-- version should MATCH as closely as possible with the underlying software -->
     <!-- Is the version a prerelease of a version? https://docs.nuget.org/create/versioning#creating-prerelease-packages -->
     <!-- Note that unstable versions like 0.0.1 can be considered a released version, but it's possible that one can release a 0.0.1-beta before you release a 0.0.1 version. If the version number is final, that is considered a released version and not a prerelease. -->
-    <version>7.21.0.23</version>
+    <version>7.21.0.2300</version>
```

Comment:
Could you please revert this change? Bumping the version number is done using AU

Reply:
Addressed in 167e20c02fb7362f6f75c7d7350814a1187bbf18.